package-pro.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs')
+const proPackage = require('./package-pro.json')
+const litePackage = require('./package.json')
+
+const combinedPackage = Object.assign({}, litePackage, {
```

Comment:
I think `_.merge` would be useful here

Reply:
I tried to avoid using packages since you can run this command before installing them.

Reply:
Oh yeah good point! Yes let's keep packages out of this